### DISCUSSION TITLE
vm-type=vz on macOS Sonoma

### DISCUSSION THREAD

**Discussion by sebbegg** [abiosoft/colima#1143] [General]
Hi there,

I'd wanted to give vz vm's a try as I'm using colima for rather heavy devcontainers.
I have colima v0.7.5 (from homebrew) installed on an m3 with macOS 14.7.

I tried editing the colima default yaml profile or to start new ones e.g. via cli:

```
❯ colima start --vm-type=vz --mount-type=virtiofs --cpu 5 --memory 12 -v --very-verbose --profile dev
TRAC[0000] cmd ["limactl" "info"]
TRAC[0000] cmd ["system_profiler" "-json" "SPHardwareDataType"]
TRAC[0000] cmd ["limactl" "list" "colima-dev" "--json"]
TRAC[0000] error retrieving running instance: instance 'colima [profile=dev]' does not exist
INFO[0000] starting colima [profile=dev]
INFO[0000] runtime: docker
INFO[0000] creating and starting ...                     context=vm
TRAC[0000] cmd ["limactl" "start" "--tty=false" "/var/folders/yn/7f7v4p1d5c96gcc4f0_rs2xm0000gn/T/colima-dev.yaml"]
> Terminal is not available, proceeding without opening an editor
> `mountType: 9p` is experimental
> Starting the instance "colima-dev" with VM driver "qemu"
...
```

I did a `colima delete (--profile dev)` before each attempt.

Whatever my changes/cli args are, colima seems to insist on using `qemu`.
The comments in `default.yaml` say `this is macOS 13 only` - should I take this literally, i.e. read it as `macOS==13` instead of `macOS >= 13` or am I missing something else?

**Comment by abiosoft** [comment]
This is quite strange. Can you retry without specifying the `--mount-type` flag?

**Reply by sebbegg** [reply]
Hi,

I think I've solved this. It wasn't the `mount-type` but I had `arch=x86_64` in the `default.yaml`.
Being on an M3 this is apparently not supported by `vz`.

It would be helpful though, it this could produce a warning at least ;)
